benchmarks/src/main/java/org/apache/druid/server/coordinator/NewestSegmentFirstPolicyBenchmark.java

```diff
@@ -127,7 +127,7 @@ public void setup()
               0,
               segmentSizeBytes
           );
-          timeline.add(segment.getInterval(), segment.getVersion(), shardSpec.createChunk(segment));
+          timeline.add(segment.getInterval(), segment.getMajorVersion(), shardSpec.createChunk(segment));
```

Comment:
After a few hacks at reviewing this PR, I'm starting to wonder if renaming `getVersion` to `getMajorVersion` worth all the noise it causes in this change. Like I think it's maybe still obvious that `getVersion` is .. more major than `getMinorVersion`? Thoughts? Am I missing a reason that this _needs_ to be renamed?

It might dramatically reduce the footprint of this PR, since a lot of the changes are to this method call. If you really prefer it, maybe it would be better to rename it in a follow-up PR that just has that change?

Reply:
Sounds good. Reverted renaming.